sample_formatted_code.do_not_merge.c

```diff
@@ -0,0 +1,562 @@
+    int a = 13;           /* comment */
+    int32_t abc = 1312;   /* comment */
+    int abcdefghikl = 13; /* comment */
+
+    // AlwaysBreakBeforeMultilineStrings
+    aaaa = "bbbb"
+           "ccc";
+
+    //--- variable init continuation behaviour
+    // Cpp11BracedListStyle impacts if space at beginning and end of brace
+    // e.g. false: Bla bla[] = { 1, 2, 3 };
+    //      true:  Bla bla[] = {1, 2, 3};
+    // Note, the different indentation of continuation line:
+    //  - Cpp11BracedListStyle:false does NOT use ContinuationIndentWidth (8)
+    //    but rather the regular IndentWidth (4)! Bug? Feature?
+    //  - Cpp11BracedListStyle:true uses ContinuationIndentWidth (8)
+    int list[] = {1, 2, 3, 4, 5, 6, 7, 8, 9};
+    int list[] = {
+            1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18};
+    int list[] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18,
+            19, 20, 21, 22};
+    PcapStats64 last = {0, 1234, 8765};
+    // trailing comma forces one-value-by-line
+    PcapStats64 trailing_comma = {
+            0,
+            1234,
+            8765,
+    };
+    struct pcap_stat current = {12, 2345, 9876, 1345};
+    struct pcap_stat current = {
+            12, 2345, 9876, 1345, 333, 444, 5555, 66666, 7777777};
+    current = {12, 2345, 9876, 1345, 333, 444, 5555, 66666, 7777777, 888888,
+            9999999};
+
+    //--- designated initializer continuation behaviour
+    // Current clang-format disables BinPacking for designated intializers when
+    // continuing on more than one line.
+    PcapStats64 last = {.ps_recv = 0, .ps_drop = 1234, .ps_ifdrop = 8765};
+    // Cpp11BracedListStyle:false puts end brace onto separate line iff
+    // continuation line can hold all intializers. Bug? Feature?
+    struct pcap_stat current = {
+            .ps_recv = 12, .ps_drop = 2345, .ps_ifdrop = 9876, .ps_what = 134};
+    pcap_stat current = {
+            .ps_recv = 12, .ps_drop = 2345, .ps_ifdrop = 9876, .ps_what = 134};
+    // One designated initializer per line if it does not fit into one
+    // continuation line as BinPacking is disabled for designated intializer.
+    struct pcap_stat current = {.ps_recv = 12,
+            .ps_drop = 2345,
+            .ps_ifdrop = 9876,
+            .ps_what = 1345,
+            .ps_more = 333};
+    current = {.ps_recv = 12,
+            .ps_drop = 2345,
+            .ps_ifdrop = 9876,
+            .ps_what = 1345,
+            .ps_more = 333};
+
+    // function call continuation
+    function_call(
+            with, many, params, that, will spill, over, eventually, iff, max);
+    function_call(with, many, params, that, will spill, over, eventually, iff,
+            we, keep, on, adding);
+}
+
+struct Bitfields {
+    // clang 11: AlignConsecutiveBitFields
+    int aaaa : 1;
+    int b : 12;
+    int ccc : 8;
+};
+
+static void wrapping_literals()
+{
+    // string literal is too long. Continuation is from "string literal start",
```

Comment:
I left this out in code-style.rst. Seemed too mundane of a detail. Only useful for initial choice of setting, imho.